src/marker.rs

```diff
@@ -29,6 +29,7 @@ impl<State, Action> View<State, Action, MapCtx, DynMessage> for Marker {
     }
 
     fn rebuild(&self, prev: &Self, _: &mut Self::ViewState, _: &mut MapCtx, e: Mut<Self::Element>) {
+        debug_assert!(matches!(e, MapChildElement::Marker(_)));
```

Comment:
@Philipp-M if i run the example and play around a bit, it panics very quickly here.
How can it be that this function is called with the wrong element?
Does this have anything to do with the ` SuperElement` or `ElementSplice` implementation?